frontend/app/routes/$lang+/_public+/apply+/$id+/adult-child/children/index.tsx

```diff
@@ -93,43 +124,69 @@ export default function ApplyFlowChildSummary() {
         <Progress aria-labelledby="progress-label" value={40} size="lg" />
       </div>
       <div className="max-w-prose">
-        <fetcher.Form method="post" aria-describedby="form-instructions-sin form-instructions" noValidate>
+        <fetcher.Form method="post" noValidate>
```

Comment:
You need `fetcher.Form` per child otherwise the `childId` will always be the first one if you have multiple children. This page will have multiple forms.

Reply:
ah, gotcha

Reply:
i split out the `fetcher.Form` into two: one for each child, and the other for the back/continue buttons since you can't have nested forms according to the console warnings

Reply:
Yup, 😀